test/runtests.jl

```diff
@@ -203,6 +217,32 @@ const url = "http://127.0.0.1:$port"
         end
     end
 
+    @testset "task backtraces" begin
+        done = Threads.Atomic{Bool}(false)
+        # Schedule some work that's known to be expensive, to profile it
+        workload() = @async begin
+            for _ in 1:200
+                if done[] return end
+                global a = [[] for i in 1:1000]
+                yield()  # yield to allow the tests to run
+            end
+        end
+
+        @testset "task_backtraces endpoint" begin
+            @static if VERSION >= v"1.10.0-DEV.0"
```

Comment:
Alternatively, we could test the on older Julia versions, we get a 501 back

Reply:
This will throw an exception, I believe.